lib.php

```diff
@@ -41,8 +41,31 @@
  * @param  array $requestparams
  */
 function ltisource_message_handler_before_launch($instance, $endpoint, $requestparams) {
+    global $CFG;
+
+    if (defined('CLI_SCRIPT') && CLI_SCRIPT) {
```

Comment:
is there any chance this would be called in CLI?

Reply:
Apart from unit tests, it's possible but unlikely.

Reply:
If there is no valid case for this to be run in CLI, I'd suggest to remove this check. Unit tests are covered below.

Reply:
Updated. Initiating tests.

Reply:
```
php vendor/bin/phpunit --testsuite mod_lti_testsuite
Moodle 4.1.14+ (Build: 20241101), fcff67b915e3765a271100f9f5d3f51043ebfd4b
Php: 7.4.33, pgsql: 16.0 (Debian 16.0-1.pgdg120+1), OS: Linux 6.8.0-49-generic x86_64
PHPUnit 9.5.28 by Sebastian Bergmann and contributors.

...............................................................  63 / 144 ( 43%)
............................................................... 126 / 144 ( 87%)
..................                                              144 / 144 (100%)

Time: 00:38.200, Memory: 133.00 MB

OK (144 tests, 539 assertions)
```